1-WebApp-OIDC/1-7-B2C-Embedded/README.md

```diff
@@ -0,0 +1,247 @@
+---
+services: active-directory
+platforms: dotnet
+author: helshabini
+level: 200
+client: ASP.NET Core Web App
+endpoint: Microsoft identity platform
+---
+# An ASP.NET Core Web app signing-in users with the Microsoft identity platform in Azure AD B2C using the Embedded sign-in experience with iframe
+
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=819)
+
+## Scenario
+
+This sample shows how to build a .NET Core MVC Web app that uses OpenID Connect to sign in users in **Azure AD B2C** using the embedded sign-in experience. It assumes you have some familiarity with **Azure AD B2C**. If you'd like to learn all that B2C has to offer, start with our documentation at [https://aka.ms/aadb2c](https://aka.ms/aadb2c).
+
+![Sign in with Azure AD](ReadmeFiles/sign-in.png)
+
+## How to run this sample
+
+To run this sample:
+
+> Pre-requisites: Install .NET Core 3.0 or later (for example for Windows) by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```powershell
+git clone https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2.git
+```
+
+> Given that the name of the sample is very long, and so are the names of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
+
+Navigate to the `"1-7-B2C-Embedded"` folder
+
+ ```Sh
+  cd "1-7-B2C-Embedded"
+  ```
+
+## Option 1 - Run the pre-configured sample
+
+1. Build the solution and run it.
+1. Open your web browser and make a request to the app. Accept the IIS Express SSL certificate if needed. Click on SignIn/Up button.
```

Comment:
@helshabini you mention that the sample is preconfigured, but it doesn't look like `appsettings.json` has those configuration parameters right now. Is that correct?

Reply:
Hi Dogan, indeed you are correct. That's simply because I don't know of an existing policy in the fabrikamb2c tenant which is enabled for JourneyFraming. Do we know if we can get to upload a policy which has JourneyFraming enabled in fabrikamb2c? In such case we can pre-fill the appsettings.json with default parameters so users can run it immediately without changing anything. I guess in such case we'll also need to enable custom domains for the fabrikamb2c tenant.

Reply:
@helshabini that would be a great improvement, let's see if we can do this. cc @mistermik who can also help us here